allennlp/modules/transformer/transformer_layer.py

```diff
@@ -203,14 +220,14 @@ def forward(
                 encoder_attention_mask,
                 output_attentions,
             )
-            attention_output = cross_attention_outputs[0]
-            outputs = (
-                outputs + cross_attention_outputs[1:]
+            attention_output = cross_attention_outputs.hidden_states
+            outputs = outputs + (  # type: ignore
+                cross_attention_outputs.attention_probs,
             )  # add cross attentions if we output attention weights
 
         intermediate_output = self.intermediate(attention_output)
         layer_output = self.output(intermediate_output, attention_output)
-        outputs = (layer_output,) + outputs
+        outputs = (layer_output,) + outputs  # type: ignore
         return outputs
```

Comment:
This thing still outputs a tuple of varying sizes, depending on what you pass in. Can we also change this one to output a `NamedTuple`?